docs/ref/contrib/postgres/search.txt

```diff
@@ -326,3 +327,42 @@ Usage example::
     ...     distance=TrigramDistance('name', test),
     ... ).filter(distance__lte=0.7).order_by('distance')
     [<Author: Katy Stevens>, <Author: Stephen Keats>]
+
+
+``TrigramWordSimilarity``
+---------------------
+
+.. class:: TrigramWordSimilarity(expression, string, **extra)
```

Comment:
`versionadded:: 4.0` annotation is missing. Also, I'd move `TrigramWordSimilarity` docs below the `TrigramSimilarity`.